x-pack/plugins/apm/public/components/app/agent_explorer_details/filter_select.tsx

```diff
@@ -0,0 +1,62 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React, { useMemo } from 'react';
+import { i18n } from '@kbn/i18n';
+import { EuiComboBox } from '@elastic/eui';
+import { EuiComboBoxOptionOption } from '@elastic/eui';
+
+interface Props {
+  title: string;
+  isLoading?: boolean;
+  options: Array<EuiComboBoxOptionOption<string>>;
+  value?: string;
+  placeholder?: string;
+  onChange: (value?: string) => void;
+  dataTestSubj?: string;
+}
+
+export function FilterSelect({
```

Comment:
You could check [SuggestionsSelect](https://github.com/elastic/kibana/blob/b31c1d6462fe1b893f0791100e257dc85121a8cf/x-pack/plugins/apm/public/components/shared/suggestions_select/index.tsx#L28)

Reply:
Thanks!